client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -69,7 +70,7 @@ function ReaderSubscriptionListItem( {
 			</div>
 			<div className="reader-subscription-list-item__options">
 				<FollowButton siteUrl={ siteUrl } followSource={ followSource } />
-				{ isFollowing && <EmailSettings /> }
+				{ isFollowing && isEmailBlocked !== true && <EmailSettings siteId={ siteId } /> }
```

Comment:
note: I don't want to rely on `! isEmailBlocked` because I'd rather show the cog during a fetch (null)